lib/macroni/Common/MLIRUtils.cpp

```diff
@@ -0,0 +1,10 @@
+#include "macroni/Common/MLIRUtils.hpp"
+#include "vast/Util/Common.hpp"
+
+namespace macroni {
+void build_region(vast::mlir_builder &bld, vast::op_state &st,
```

Comment:
There is exactly the same `vast::build_region` in `vast/Util/Region.hpp`?

Reply:
Yes, when I tried linking to VASTUtil though I received many errors saying `use of undeclared identifier 'gap'`, so it was easier for me to just redefine it for Macroni.

Reply:
Ok I think I figured it out.
So the vast header that declares `build_region()` indirectly includes the gap header `gap/coro/generator.hpp`, and the declarations in this header are nested under `#ifdef GAP_ENABLE_COROUTINES`.
I turned on this macro project-wide and the problem went away.
I should be able to add that definition to a specific target to achieve the same result, but I'm not sure which target to add it to yet.